cmd/kube-apiserver/app/server.go

```diff
@@ -402,6 +408,9 @@ func buildGenericConfig(
 	if lastErr = s.APIEnablement.ApplyTo(genericConfig, master.DefaultAPIResourceConfigSource(), legacyscheme.Scheme); lastErr != nil {
 		return
 	}
+	if connectivityService, lastErr = s.NetworkProxy.ApplyTo(genericConfig); lastErr != nil {
```

Comment:
`ApplyTo` generally sets values on the `genericConfig`, it doesn't usually return something extra.  It seems like this should live either on the kube-apiserver or the genericapiserver.  Given that other servers are likely to want to contact external services, I think it makes sense on a genericapiserver config.

Reply:
+1

Reply:
Fixed.